operator/controllers/common/config/config.go

```diff
@@ -0,0 +1,36 @@
+package config
+
+import (
+	"sync"
+	"time"
+)
+
+const defaultKeptnAppCreationRequestTimeout = 30 * time.Second
+
+//go:generate moq -pkg fake -skip-ensure -out ./fake/config_mock.go . IConfig:MockConfig
+type IConfig interface {
+	SetCreationRequestTimeout(value time.Duration)
+	GetCreationRequestTimeout() time.Duration
+}
+
+type Config struct {
```

Comment:
Since we have the `KeptnConfig`, maybe we should rename this to avoid confusion
```suggestion
type ControllerConfig struct {
```

Furthermore, this field comes from the `KeptnConfig` CRD. Will the wiring up with that reconciler be done in a follow-up?

Reply:
yes, right now this config will always return the default of 30s, as described in the ticket - wiring up with the config reconciler will be done in a separate PR